textproc/glow/Portfile

```diff
@@ -29,6 +29,16 @@ build.args-append   -ldflags=\"-X 'main.Version=${version}'\"
 
 destroot {
     xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
+    # All supported shells: bash, fish, powershell, zsh
+    set comp_path(bash) ${destroot}${prefix}/share/bash-completion/completions/${name}
+    set comp_path(fish) ${destroot}${prefix}/share/fish/completions/${name}.fish
+    set comp_path(zsh)  ${destroot}${prefix}/share/zsh/site-functions/_${name}
+    foreach shell {bash fish zsh} {
+        xinstall -d [file dirname $comp_path(${shell})]
+        system -W ${worksrcpath} "./${name} completion ${shell} > $comp_path(${shell})"
```

Comment:
Typically you want to run the executable in a `build` or `post-build` block generating the completions to some output path in `${worksrcpath}`, and _then_ installing from that location in `destroot`.

Reply:
Thought about doing that but it would require 2 foreach loops and a few more variables so I didn't bother.